src/i_video.c

```diff
@@ -1327,6 +1329,8 @@ static void I_InitGraphicsMode(void)
         flags |= SDL_WINDOW_BORDERLESS;
     }
 
+    SDL_SetHint(SDL_HINT_MOUSE_RELATIVE_SCALING, "0");
```

Comment:
Could you add a comment here, please?

Reply:
I'm not sure if I actually want to keep this. I may change the SDL mouse reading functions in `I_ReadMouse()` which would eliminate the need for this hint.